staging/vhost-device-sound/src/audio_backends/pipewire.rs

```diff
@@ -336,6 +338,7 @@ impl AudioBackend for PwBackend {
                             }
                             let p = &mut slice[0..n_bytes];
                             if avail <= 0 {
+                                // SAFETY: safe to zero-initialize the pointer
```

Comment:
I think it's safe here more because we know the pointer is valid and points to a memory area of  `n_bytes` bytes.

Reply:
Maybe also move the `// pad with silence` inside the block, since it is not clear if it is related to safety or not.

Reply:
ok noted thanks